setup/db/create-database-simulator.sql

```diff
@@ -20,8 +20,8 @@ DROP DATABASE IF EXISTS `simulator`;
 
 CREATE DATABASE `simulator`;
 
-GRANT ALL ON simulator.* to cloud@`localhost` identified by 'cloud';
-GRANT ALL ON simulator.* to cloud@`%` identified by 'cloud';
+GRANT ALL ON simulator.* to cloud@`localhost`;
+GRANT ALL ON simulator.* to cloud@`%`;
```

Comment:
@shwstppr I assume a password for this user is set elsewhere?

Reply:
@andrijapanicsb User `cloud` with the password will already be created in create-database.sql https://github.com/apache/cloudstack/pull/4068/commits/b895cf98f3927b8fbae7c927367e2c1d3bb68e0b#diff-ca79fa3d84677eec693e98f41aca3a0eR58-R59